Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -69,21 +73,43 @@ class TopSitesPanel: UIViewController, UICollectionViewDelegate, HomePanel {
     private var collection: TopSitesCollectionView!
     private var dataSource: TopSitesDataSource!
     private let layout = TopSitesLayout()
+    private lazy var longPressGesture: UILongPressGestureRecognizer = {
+        return UILongPressGestureRecognizer(target: self, action: "SELdidLongPress")
+    }()
+    private var closeButtonDidAnimateMap = [Int: Bool]()
 
-    var profile: Profile! {
+    var editMode : Bool = false {
```

Comment:
We use "editing mode" to refer to the state when the user has tapped the URL bar and the home overlay is shown (as opposed to the user simply being on about:home), so calling this `editMode` could be confusing. Maybe change this to something like `editingThumbnails`?